src/util/encoding_rlp.hpp

```diff
@@ -13,129 +13,215 @@ namespace taraxa::util::encoding_rlp {
 using namespace std;
 using namespace dev;
 
+using RLPEncoderRef = RLPStream&;
+struct RLPDecoderRef {
+  RLP const& target;
+  RLP::Strictness strictness;
```

Comment:
also missing *_ suffix

Reply:
https://github.com/Taraxa-project/taraxa-node/pull/843#discussion_r648598852

Reply:
Is this a misuse of `struct`? If not, then the naming is correct as per the Google naming convention. @JakubFornadel @MatusKysel please confirm

Reply:
I do no think that this is misuse, so this should be alright as it is